sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+} from "@nestjs/common";
+import { ApplicationService } from "../../services";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import BaseController from "../BaseController";
+import { CreateApplicationDto } from "./models/application.model";
+
+@Controller("application")
+export class ApplicationController extends BaseController {
+  constructor(private readonly applicationService: ApplicationService) {
+    super();
+  }
+
+  @Get(":id/data")
```

Comment:
:id is enough i guess
Consider removing the word "data" from the path

Reply:
Yes, it is enough. I added the "data" to make it explicit that we are getting only part of the application, the "data" part.
In the future, if we are going to add another get to retrieve another application fields this could be kept as is.